projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/linked-cases/components/linked-cases-table/linked-cases-from-table.component.ts

```diff
@@ -0,0 +1,101 @@
+import { AfterViewInit, Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { CaseField } from '../../../../../domain/definition';
+import { CaseView } from '../../../../../domain';
+import { CasesService } from '../../../../case-editor/services/cases.service';
+import { CaseLinkResponse, LinkedCasesResponse } from '../../domain/linked-cases.model';
+import { ActivatedRoute } from '@angular/router';
+import { LovRefDataModel } from '../../../../../services/common-data-service/common-data-service';
+import { Observable } from 'rxjs';
+import { LinkedCasesService } from '../../services';
+
+@Component({
+  selector: 'ccd-linked-cases-from-table',
+  templateUrl: './linked-cases-from-table.component.html',
+  styleUrls: ['./linked-cases-from-table.component.scss']
+})
+
+export class LinkedCasesFromTableComponent implements OnInit, AfterViewInit {
+
+  private static readonly CASE_NAME_MISSING_TEXT = 'Case name missing';
+
+  @Input()
+  public caseField: CaseField;
+
+  @Output()
+  public notifyAPIFailure: EventEmitter<boolean> = new EventEmitter(false);
+  public caseDetails: CaseView;
+  public parentUrl: string;
+  public isLoaded: boolean;
+  public getLinkedCasesResponse: CaseLinkResponse[] = [];
+  public linkedCaseReasons: LovRefDataModel[];
+
+  public caseId: string;
+  public showHideLinkText = 'Show';
+  public noLinkedCases = true;
+  public isServerError = false;
+
+  constructor(
+    private readonly route: ActivatedRoute,
+    private readonly casesService: CasesService,
+    public readonly linkedCasesService: LinkedCasesService,
```

Comment:
@johnbenjamin-hmcts - query use of `public` vs `private` for the other injections (see similar comment for another case above)

Reply:
@Daniel-Lam - It was odd to keep it public to access in the HMTL. I have done the code changes accordingly. These were already existing code.